transport/tlscommon/tls_config_test.go

```diff
@@ -763,10 +764,11 @@ func genTestCerts(t *testing.T) map[string]*x509.Certificate {
 		// We write the certificate to disk, so if the test fails the certs can
 		// be inspected/reused
 		certPEM := new(bytes.Buffer)
-		pem.Encode(certPEM, &pem.Block{
+		err = pem.Encode(certPEM, &pem.Block{
 			Type:  "CERTIFICATE",
 			Bytes: cert.Leaf.Raw,
 		})
+		require.Errorf(t, err, "failed to encode certificste to PEM")
```

Comment:
There is nothing to be formatted here:
```suggestion
		require.Error(t, err, "failed to encode certificste to PEM")
```